packages/shared/routes/onboarding/views/ledger-setup/views/AccountIndexView.svelte

```diff
@@ -2,10 +2,8 @@
     import { onDestroy, createEventDispatcher } from 'svelte'
```

Comment:
I am not sure we even need this file anymore since it and a few others were apart of migration. Maybe we can make a cleanup task to completely remove any migration logic from the app? I know there even used to be an advanced setting related to it, so probably worth looking into.

Reply:
https://github.com/iotaledger/firefly/issues/4392